doctor/templates/doctor/doctor_details.html

```diff
@@ -1,23 +1,101 @@
-<script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+    crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js"
+    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+    crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js"
+    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+    crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
-<div id="alert-container" class="alert alert-success alert-dismissible fade show fixed-alert" role="alert" style="display: none;">
-    <strong><span id="alert_message">Hello There</span></strong>
-    <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-        <span aria-hidden="true">&times;</span>
-    </button>
-</div>
 
-<h1 id="doctor_heading" class="display-4">Doctor John Doe</h1>
 
-<div class="details-main-wrapper">
-    <button id="btn_book_online_consultation" class="formbold-btn" data-toggle="modal" data-target="#book_appointment_modal">Book Online Appointment</button>
-</div>
+<div class="page">
 
-{% include "doctor/online_appointment_form.html" %}
+    <div id="alert-container" class="alert alert-success alert-dismissible fade show fixed-alert" role="alert"
+        style="display: none;">
+        <strong><span id="alert_message">Hello There</span></strong>
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
 
-<script src="{% static 'doctor/javascript/doctor_details.js' %}"></script>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Doctor {{object.first_name}} {{doctor.last_name}}</title>
+    </head>
+
+    <div class="header">
+        <!-- <img src="logo.png" alt="logo" class="logo"> -->
+        <span>Doctor {{object.first_name}} {{doctor.last_name}}</span>
+        <nav>
+            <a href="#">Search</a>
```

Comment:
hey can you replace search with Home instead? Cuz it may be misinterpreted that you wanna search this particular doctor page. Later we can connect Home to the home page.

Reply:
Of course, this header is just used as a template for the future implementation, I'm not deciding what links we need to put here, we could talk about this in next scrum meeting